tools/fast/series/global_sleep_delay.go

```diff
@@ -0,0 +1,17 @@
+package series
+
+import (
+	"time"
+
+	"github.com/filecoin-project/go-filecoin/mining"
+)
+
+// GlobalSleepDelay is the time value that is used to wait for block processing
+// in series
+var GlobalSleepDelay = mining.DefaultBlockTime
+
+// SleepDelay is a helper method to make sure people
+// don't call `time.Sleep` themselves in series.
+func SleepDelay() {
+	time.Sleep(GlobalSleepDelay)
+}
```

Comment:
I expect this to possibility be controversial. One thought I had to help remove the global was to change series to be a package that you initiate, and maybe provide an environment too. That would also help with managing the passing of some redundant information in some places, and would be a good way to remove this global.

Reply:
i dont find it so onerous, but what you suggest sounds better, but more work